CHANGELOG.md

```diff
@@ -42,7 +42,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   - Remove the option for output type.(`-o`) The associated functionality
     is now deprecated.
 - Changed configuration field names.
-  - `roots` to `root` to handle using a single root.
+  - `roots` to `ca_certs`.
```

Comment:
I don't think this line can change, because it is already released as 0.19.0. Could you add a new entry in Unreleased section?

Reply:
Done!